scripts/get_resource_names.go

```diff
@@ -0,0 +1,229 @@
+// get_resource_names collects the resource names from the test files in test-artifacts
```

Comment:
Can this script be named `get_test_resource_names.go`, so it's instantly clear it's about testing? Actual, as we have non testing items in this folder as well.

Reply:
Renamed